src/core/Configuration.py

```diff
@@ -129,13 +129,25 @@ def hostname(self):
         Indicates if we are in a development mode (= clean database before mount for example) or not.
     """
     def is_development(self):
-        return self.conf['development']
+        return False if 'development' not in self.conf else self.conf['development']
 
     """
         The chunk size in gridfs. Value must be between 1 and 15MB maximum (to allow overhead of other fields)
     """
     def chunk_size(self):
         chunk_size = self.conf['mongo']['chunk_size']
-        if chunk_size < 1 or chunk_size > 15*1024*1024:
-            raise ValueError('Invalid chunk size, must be between 1 and '+str(15*1024*1024)+' bytes.')
+        if chunk_size < 1 or chunk_size > 15 * 1024 * 1024:
+            raise ValueError('Invalid chunk size, must be between 1 and ' + str(15 * 1024 * 1024) + ' bytes.')
         return chunk_size
+
+    """
+        Sets the default mode of the root node in mongofs
+    """
+    def default_root_mode(self):
+        return 0o755 if 'default_root_mode' not in self.conf else int(self.conf['default_root_mode'], 8)
```

Comment:
This is not easily readable. Please use something like self.conf.get('default_root_mode', ...) or write in multiple lines.